hyperspy/roi.py

```diff
@@ -607,6 +607,56 @@ def _top_changed(self, old, new):
         else:
             self.update()
 
+    @property
+    def width(self):
+        return self.right - self.left
```

Comment:
And possibly a mention in class docstrings as they're part of the public interface.